Projects/Foundations/Schemas/Docs-Nodes/E/Execution/Execution-Algorithm/execution-algorithm.json

```diff
@@ -131,7 +134,8 @@
                 {
                     "language": "TR",
                     "text": "Yürütme Algoritması Yapılandırması",
-                    "updated": 1654755849909
+                    "updated": 1654392450016,
+                    "style": "Title"
```

Comment:
Just out of interest did you translate this file in Superalgos itself and add the style "text" in the translation or did you use the automatic tool?

Reply:
I used the Superalgos Translator tool. And then I checked the translations myself. Finally, I created a PR.